### PR TITLE
All users can create tags

### PR DESCRIPTION
**Pull request by Raemon**

### CODE REVIEW

File: packages/lesswrong/components/sunshineDashboard/SunshineNewTagsItem.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    fragmentName: 'SunshineTagFragment',
+  });
+
+
+
+  const handleApprove = () => {
+    updateTag({
+      selector: { _id: tag._id},
+      data: {
+        reviewedByUserId: currentUser!._id,
```
Comment (Discordius) [inline packages/lesswrong/components/sunshineDashboard/SunshineNewTagsItem.tsx:55]: Why are you using an exclamation point here? I think you should add a null-check to the beginning of this function if you want the type-system to do what you want.

Comment (Raemon) [reply packages/lesswrong/components/sunshineDashboard/SunshineNewTagsItem.tsx:55]: That looks like a straight-up error, probably not supposed to be there at all. But, adding a null check seems good.

Comment (Raemon) [reply packages/lesswrong/components/sunshineDashboard/SunshineNewTagsItem.tsx:55]: Oh, this was copy-pasted from other Sunshine items. I think it's plausible they should all change, but I think I'd prefer to do that in a separate pass? I don't actually know what the exclamation point does.

Comment (Raemon) [reply packages/lesswrong/components/sunshineDashboard/SunshineNewTagsItem.tsx:55]: (looks like Jim added them to other Sunshine elements, you may want to chat with Jim and get into agreement on preferred approach here)

**Pull request merged**